app/views/entities/promo.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# presenter = EntityPresenter.new(@entity)
+
+json.url portal_entity_path(@entity.uri, format: 'html')
+json.title false
```

Comment:
why set this? Is there anything more applicable that can go into the title than 'false'?